src/resources/ble_esp32.cc

```diff
@@ -233,6 +233,21 @@ class BleReadWriteElement : public BleErrorCapableResource {
     return unvoid_cast<BleReadWriteElement*>(arg)->_on_access(ctxt);
   }
 
+  static uint16_t mbuf_total_len(os_mbuf* om) {
```

Comment:
You could make this work for the `om == null` case quite easily and the code would be shorter. You would get one extra null check per call though.